gtclang/test/unit-test/Frontend/TestIRSplitting.cpp

```diff
@@ -41,6 +41,11 @@ TEST(IRSplittingTest, CacheTest) {
       .split("PassSetCaches/IJCacheTest02.cpp", {"-freport-pass-set-caches"});
 }
 
+TEST(IRSplittingTest, IntervalPartitioning) {
```

Comment:
can you explain what this part is?

Reply:
Just code I have been using to generate different IR levels without running through the entire toolchain. The executable splitting PR will render it unnecessary.